modules/cms/controllers/ThemeOptions.php

```diff
@@ -143,4 +151,22 @@ protected function findThemeObject($name = null)
 
         return $theme;
     }
+
+    /**
+     * Register language namespace if available for the theme
+     *
+     * @param string $dirName
+     *
+     * @throws ApplicationException
```

Comment:
```suggestion
     * @param string $dirName
     * @throws ApplicationException
```
Also when would an application exception be thrown?

Reply:
It is being thrown in `findThemeObject` method. And as long as it is not caught here exception is bubbling and this method automatically starts throwing the same.